administration/src/components/applications/ApplicationsController.tsx

```diff
@@ -12,7 +12,7 @@ const ApplicationsController = (props: { region: Region }) => {
   })
   if (loading) return <Spinner />
   else if (error || !data) return <ErrorHandler refetch={refetch} />
-  else return <ApplicationsOverview applications={data.applications} />
+  else return <ApplicationsOverview applications={data.applications.filter(application=>!application.withdrawalDate)} />
```

Comment:
should the application be filtered or just labeled in the overview and remove the delete action?